alf/environments/utils.py

```diff
@@ -73,6 +73,8 @@ def _env_constructor(env_load_fn, env_name, batch_size_per_env, seed, env_id):
 @alf.configurable
 def create_environment(env_name='CartPole-v0',
                        env_load_fn=suite_gym.load,
+                       eval_env_load_fn=None,
+                       create_eval_env=False,
```

Comment:
The argument name is misleading. It can be interpreted as "creating an additional eval env". Can be changed to just "for_evaluation".

Reply:
changed as suggested